types/kakaomaps/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false,
```

Comment:
Please don't disable this rule.  It's just asking you to omit the patch version from the header because the patch version is computed when these types are published.

Reply:
Thanks, I fixed this issue as you recommended